Samples/src/CHSHGame/Driver.cs

```diff
@@ -0,0 +1,91 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Microsoft.Quantum.Samples.CHSHGame
+{
+    using System;
+    using Microsoft.Quantum.Simulation.Simulators;
+
+    /// <summary>
+    /// The classical driver for the quantum computation.
+    /// </summary>
+    public class Driver
+    {
+        /// <summary>
+        /// Main entry point to the program.
+        /// </summary>
+        /// <param name="args">Command-line parameters.</param>
+        static void Main(string[] args)
+        {
+            const int trialCount = 10000;
+            Random generator = new Random();
+            using (QuantumSimulator sim = new QuantumSimulator())
+            {
+                int classicalWinCount = 0;
+                int quantumWinCount = 0;
+                for (int i = 0; i < trialCount; i++)
+                {
+                    bool aliceBit = GetRandomBit(generator);
+                    bool bobBit = GetRandomBit(generator);
+                    bool aliceMeasuresFirst = GetRandomBit(generator);
+                    bool classicalXor =
+                        !PlayClassicalStrategy(aliceBit, bobBit);
+                    bool quantumXor =
+                        !PlayQuantumStrategy.Run(
+                            sim,
+                            aliceBit,
+                            bobBit,
+                            aliceMeasuresFirst).Result;
+
+                    if ((aliceBit && bobBit) == classicalXor)
+                    {
+                        classicalWinCount++;
+                    }
+
+                    if ((aliceBit && bobBit) == quantumXor)
+                    {
+                        quantumWinCount++;
+                    }
+                }
+
+                Console.WriteLine(
+                    "Classical success rate: "
+                    + classicalWinCount / (float)trialCount);
+                Console.WriteLine(
+                    "Quantum success rate: "
+                    + quantumWinCount / (float)trialCount);
+
+                if (quantumWinCount > classicalWinCount)
+                {
+                    Console.WriteLine("SPOOKY");
```

Comment:
Could we put a more informative message in this case.  
```suggestion
                    Console.WriteLine("The quantum success rate exceeded the classical success rate!");
```